<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QFrame" name="menuLateral">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>181</width>
     <height>601</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>181</width>
     <height>601</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>181</width>
     <height>601</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(145, 0, 72);

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnLucesMediasDS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>171</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(145, 0, 72);
border-top-left-radius: 15px;
border-bottom-left-radius: 15px;
	color: rgb(255, 255, 255);

}

QPushButton:hover:!pressed{

	background-color: rgb(208, 163, 47);
	color: rgb(145, 0, 72);
}

QPushButton:pressed
{
background-color: rgb(31, 33, 40);     
}</string>
    </property>
    <property name="text">
     <string>Luces Medias</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNeblinerosDS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>171</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(145, 0, 72);
border-top-left-radius: 15px;
border-bottom-left-radius: 15px;
	color: rgb(255, 255, 255);

}

QPushButton:hover:!pressed{

	background-color: rgb(208, 163, 47);
	color: rgb(145, 0, 72);
}

QPushButton:pressed
{
background-color: rgb(31, 33, 40);     
}</string>
    </property>
    <property name="text">
     <string>Neblineros</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnParqueoDS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>171</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(145, 0, 72);
border-top-left-radius: 15px;
border-bottom-left-radius: 15px;
	color: rgb(255, 255, 255);

}

QPushButton:hover:!pressed{

	background-color: rgb(208, 163, 47);
	color: rgb(145, 0, 72);
}

QPushButton:pressed
{
background-color: rgb(31, 33, 40);     
}</string>
    </property>
    <property name="text">
     <string>Parqueo</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDireccionalesDS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>171</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(145, 0, 72);
border-top-left-radius: 15px;
border-bottom-left-radius: 15px;
	color: rgb(255, 255, 255);

}

QPushButton:hover:!pressed{

	background-color: rgb(208, 163, 47);
	color: rgb(145, 0, 72);
}

QPushButton:pressed
{
background-color: rgb(31, 33, 40);     
}</string>
    </property>
    <property name="text">
     <string>Direccionales</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLucesAltasDS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>171</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(145, 0, 72);
border-top-left-radius: 15px;
border-bottom-left-radius: 15px;
	color: rgb(255, 255, 255);

}

QPushButton:hover:!pressed{

	background-color: rgb(208, 163, 47);
	color: rgb(145, 0, 72);
}

QPushButton:pressed
{
background-color: rgb(31, 33, 40);     
}</string>
    </property>
    <property name="text">
     <string>Luces Altas</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLuzSalonDS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>171</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(145, 0, 72);
border-top-left-radius: 15px;
border-bottom-left-radius: 15px;
	color: rgb(255, 255, 255);

}

QPushButton:hover:!pressed{

	background-color: rgb(208, 163, 47);
	color: rgb(145, 0, 72);
}

QPushButton:pressed
{
background-color: rgb(31, 33, 40);     
}</string>
    </property>
    <property name="text">
     <string>Luz de Sal√≥n</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCerrar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>841</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>841</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(145, 0, 72);</string>
   </property>
   <property name="text">
    <string>UNIVERSIDAD INTERNACIONAL DEL ECUADOR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>841</width>
     <height>551</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Inicio">
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>190</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pgLucesAltas">
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>781</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 0, 72);
border-radius: 15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Luces altas</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>781</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(36, 39, 49);
border-radius: 15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>341</width>
        <height>241</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>341</width>
        <height>241</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>341</width>
        <height>241</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>img/fontal.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>100</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="pgLucesMedias">
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>741</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(36, 39, 49);
border-radius: 15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Luces Medias</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
